ui/src/app/pages/analysis/interactive-notebook.tsx

```diff
@@ -200,8 +207,8 @@ export const InteractiveNotebook = fp.flow(
         );
         onRuntimeReady();
       } catch (e) {
-        this.setState({ userRequestedExecutableNotebook: false });
         if (e instanceof InitialRuntimeNotFoundError) {
```

Comment:
Are there any other errors that could prevent the user from proceeding?

Reply:
Thanks for pointing this out, I actually didn't mean to leave this change in. I wound up going with an approach that didn't depend on this boolean. Instead, the error has priority in rendering the top-bar status.

Reply:
Let me know tomorrow if this resolved your question here - I'm not in a rush to merge this

Reply:
This seems good to me.